### PR TITLE
Add formatting guidelines to CONTRIBUTING.md

### LINKED ISSUE #97: Add .editorconfig to ensure common formatting
See [here](http://editorconfig.org/).

Or, we could simply add details to `CONTRIBUTING.md`.

### PR DESCRIPTION
**Pull request by enocom**
Fixes #97

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -31,6 +31,14 @@ Participation in this project comes under the [Contributor Covenant Code of Cond
 - Once review has occurred, please rebase your PR down to a single commit. This will ensure a nice clean Git history.
 - Finally - *Thanks* for considering submitting code to Agones!
 
+## Formatting
+
+When submitting pull requests, make sure to do the following:
+
+- Format all Go code with [gofmt](https://golang.org/cmd/gofmt/).
```
Comment (markmandel) [inline CONTRIBUTING.md]: `gofmt` and/or `goimport` ?

Comment (enocom) [reply CONTRIBUTING.md]: Hmm. Technically `gofmt`, but you'd be mad to write your own import statements and not use `goimports`.

Comment (enocom) [reply CONTRIBUTING.md]: How about another sentence after this one that says, "Many people use `goimports` which adds import statements automatically while also running `gofmt`"?

Comment (markmandel) [reply CONTRIBUTING.md]: SGTM.

**Comment by markmandel** [comment]
Content LGTM - I'll repeat my earlier question, I think it got missed:

> Curious question - #97 does mention a .editorconfig but we don't have an .editorconfig , even though this is closing that ticket.
>
> Should we have a .editorconfig?

**Comment by enocom** [comment]
@markmandel Since opening the issue, I have realized `gofmt` does what `.editorconfig` hopes to do. So, I think we're good without it.

**Pull request merged**